Generate `.obdata` file for use with FlexBV (#16)
whitequark/kicad-boardview: README.md

This will read the PCB layout from `example/example.kicad_pcb` and write it to `example/example.brd`.

> Optionally, you can also use this script to export a `.obdata` file for your boards for viewing component values and package sizes using the [OpenBoardData project][OpenBoardData].

Reviewer: ```suggestion
Optionally, you can also use this script to export an `.obdata` file for your boards that contain component values and package sizes in the [OpenBoardData format][OpenBoardData].
```

> [FlexBV]: https://pldaniels.com/flexbv

Reviewer: So actually when I suggested mentioning FlexBV I did not realize that it's a commercial application. I think you should only refer to https://openboarddata.org in the README as that is the community project.

Author: Okay perfect. For the lower section should I specify that the screenshots are from the free version?

Reviewer: No references to FlexBV at all, only openboarddata.org.

Author: So do I remove the screenshot entirely or just the mention that it is from FlexBV?

Reviewer: Remove it entirely.
